### DISCUSSION TITLE
🙌 DAI 2024-2025 - Practical work 3 - Leonard Cseres, Aude Laydu

### DISCUSSION THREAD

**Discussion by leonardcser** [heig-vd-dai-course/heig-vd-dai-course#547] [Show and tell]
Purrchase est une API qui permet de découvrir des chats, créer, modifier et supprimer des posts de chat. 

Une authentification basique sera implémentée.

# Purrchase

### User

- POST /login
- POST /logout
- POST /register

- GET /profile/<id>

### Cats

- GET /cats/<id>
- POST /cats
- PUT /cats/<id>
- DELETE /cats/<id>

- GET /cats?color=..&age=...&userId=...

**Comment by ludelafo** [comment]
Tout bon pour nous.

**Comment by eau2** [comment]
Url du repo : https://github.com/HEIGVD-S3-DAI/purrchase
hash du dernier commit à considérer pour le rendu : 789cac22ad7cf2a3f9699d22486e981a788c8eb2
Bonne soirée :)
